tests/util.py

```diff
@@ -19,3 +22,11 @@ def _add_rec(db, *args, **kw):
 
 def _tagset(s):
     return set(x for x in str(s or '').lower().split(',') if x)
+
+def append(buffer, text):
+    pos = buffer.tell()
+    try:
+        buffer.seek(0, os.SEEK_END)
+        return buffer.write(text)
+    finally:
+        buffer.seek(pos)
```

Comment:
This isn't really used as of yet, but it could come in handy when testing the `stdin` fixture (…which could've been placed here as well, but `pylint` doesn't like imported fixtures)